resources/definitions/fdmprinter.def.json

```diff
@@ -1089,6 +1097,14 @@
                     "value": "'lines' if infill_sparse_density > 25 else 'grid'",
                     "settable_per_mesh": true
                 },
+                "infill_angles":
+                {
+                    "label": "Infill Line Directions",
+                    "description": "A list of integer line directions to use. Elements from the list are used sequentially as the layers progress and when the end of the list is reached, it starts at the beginning again. The list items are separated by commas and the whole list is contained in square brackets. Default is an empty list which means use the traditional default angles (45 and 135 degrees for the lines and zig zag patterns and 45 degrees for all other patterns).",
+                    "type": "[int]",
+                    "default_value": "[ ]",
```

Comment:
This setting should not be enabled for the Cubic Subdivision infill type. I don't think you made the changes to Cubic Subdivision infill for it to apply the infill angles (and I don't think you should).